### ISSUE TITLE
[Bug] site search crashes browser

### ISSUE THREAD

**Issue by jhnc** [duckduckgo/Android#3993]
### Describe the bug

submitting a site search from a form makes the browser crash

### How to Reproduce

I added this code to a webpage on my site (substitute `example.org` to test):

```
<form method="get" action="https://duckduckgo.com/"
  target="_blank" rel="noopener noreferrer">
    <input type="hidden" name="sites" value="example.org"/>
    <input type="text" name="q" maxlength="200" onfocus="this.select()" />
    <input type="submit" value="Search" />
</form>
```

If I load the webpage, type in a search query, and submit, then after a couple of seconds, the browser crashes ("DuckDuckGo has stopped").

### Expected behavior

I expect a webpage to open with the results of the search, as it does when the same webpage is tested using other browsers.

### Environment

```markdown
- DDG App Version: 5.181.1 (51811000)
- Device: Samsung Galaxy S7
- OS: Android 8.0.0
```

**Comment by VISHNUONE8** [comment]
Hey @jhnc I tried to reproduce the issue in the latest version of ddg 5.197.1 and I was unable to reproduce it.
Could you please try it on the latest version and check if the issue still persists or not ?
Thanks,
Vishnu

**Comment by jhnc** [comment]
Looks good on 5.198.0 which I am now using.

**Comment by aitorvs** [comment]
We had a regression that could cause that crash, fixed in 5.198.0 indeed.

Closing this issue for now. Thanks a lot for reporting.
